unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Scene/MessagingBus.cs

```diff
@@ -116,7 +116,7 @@ public MessagingBus(MessagingBusType type, IMessageProcessHandler handler, Messa
             this.type = type;
             this.owner = owner;
             this.pendingMessagesCount = 0;
-            sceneController = SceneController.i;
+            sceneController = Environment.i.sceneController;
```

Comment:
I notice that this `sceneController` private variable is not being used in this class (only to be assigned in the constructor). We should remove it.

Reply:
Will do!

Reply:
Thanks! The only thing left to do is to make this change, so I'm going to approve the PR.

Reply:
feedback applied!